thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: "Invalid parameter `count`: 10. Should be larger than zero, a multiple of two and less than half the population size."', libcore/result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.

real	0m0.555s
user	0m1.663s
sys	0m0.230s
